common/xbps-src/shutils/build_dependencies.sh

```diff
@@ -406,8 +406,8 @@ install_pkg_deps() {
 
     if [[ ${host_binpkg_deps} ]]; then
         if [ -z "$XBPS_QUIET" ]; then
-            # normal messages in bold
-            [[ $NOCOLORS ]] || printf "\033[1m"
+            # normal messages in bright bold white
+            [[ $NOCOLORS ]] || printf "\033[97;1m"
             echo "=> $pkgver: installing host dependencies: ${host_binpkg_deps[@]} ..."
```

Comment:
can these echos be replaced with `msg_normal`?

Reply:
I think they should, considering that there's `msg_{normal,error}` in that same function and that those two lines use `echo`s to print the message instead of `printf`s

Reply:
They have been converted now.